manager/oracles.js

```diff
@@ -637,39 +637,40 @@ const instagram = async (UserId, link) => {
         var fbPage = await FbPage.findOne({
             instagram_username: instagramUserName,
         })
-
         if (fbPage && fbPage.instagram_id) {
             var instagram_id = fbPage.instagram_id
             var fbProfile = await FbProfile.findOne({ UserId: UserId }).lean()
             if (fbProfile) {
                 var accessToken = fbProfile.accessToken
                 var mediaGetNewAccessToken = `https://graph.facebook.com/${oauth.facebook.fbGraphVersion}/oauth/access_token?grant_type=fb_exchange_token&client_id=${process.env.APPID}&client_secret=${process.env.APP_SECRET}&fb_exchange_token=${accessToken}`
-                var resMediaAccessToken = (await rp.get(mediaGetNewAccessToken)).data
-                var media =
-                    'https://graph.facebook.com/' +
-                    oauth.facebook.fbGraphVersion +
-                    '/' +
-                    instagram_id +
-                    '/media?fields=like_count,shortcode,media_url&limit=50&access_token=' +
-                    resMediaAccessToken.access_token
-                var resMedia = (await rp.get(media)).data
-                var data = resMedia.data
+                var resMediaAccessToken = (await rp.get(mediaGetNewAccessToken))
```

Comment:
In reviewing the code, it's noticed that there are some variables that could be more descriptively named to better convey their purpose or the data they hold.

A variable name should give a clear, immediate understanding of what data it contains and how it is used. For example, `perf` could be renamed to something like `postPerformanceData` for more clarity.

```javascript
// Instead of this:
var perf = { shares: 0, likes: 0, views: 0, media_url: '' }

// Use this:
var postPerformanceData = { shares: 0, likes: 0, views: 0, media_url: '' }
```

Also, the convention for JavaScript variable and constant names is camel case, not snake case. For example, `instagram_id` should be `instagramId`. This is in line with the widely accepted JavaScript naming conventions and can improve code readability.

```javascript
// Instead of this:
var instagram_id = fbPage.instagram_id

// Use this:
var instagramId = fbPage.instagramId
```

Remember, consistently using clear, descriptive names for variables and constants is one of the most effective ways to make your code self-documenting and easier to understand for other developers.